packages/google_sign_in/google_sign_in_android/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: google_sign_in_android
 description: Android implementation of the google_sign_in plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/google_sign_in/google_sign_in_android
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+google_sign_in%22
-version: 6.0.1
+version: NEXT
```

Comment:
A version bump is not needed here as per https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#updating-a-changelog-that-has-a-next.

Reply:
This does need a version bump; it changes production code. The CHANGELOG should not use `NEXT`.